src/main/kotlin/de/triplet/gradle/play/PlayPublisherPluginExtension.kt

```diff
@@ -0,0 +1,27 @@
+package de.triplet.gradle.play
+
+import java.io.File
+
+open class PlayPublisherPluginExtension {
+    var serviceAccountEmail: String? = null
```

Comment:
Yes. Using the old approach should trigger a warning and refer to the new playAccountConfig